client/src/pages/UserProfile.jsx

```diff
@@ -10,7 +10,7 @@ export function UserProfile(props) {
 
   useEffect(() => {
     if (userData._id) {
-      fetch(`/reviews/${userData._id}`)
+      fetch('/reviews/${userData._id}')
```

Comment:
the string inside the fetch request needs a template literal (between backticks '') to allow for variable referencing.